src/lib.rs.in

```diff
@@ -140,7 +155,7 @@ impl Handler for WelcomeHandler {
     fn handle(&self, _: &mut Request) -> IronResult<Response> {
         json_response(&ObjectBuilder::new()
             .insert("ladaemon", "Welcome")
-            .insert("version", &self.app.version)
+            .insert("version", env!("CARGO_PKG_VERSION"))
```

Comment:
I'd prefer to have `VERSION` in lib and refer to it from `main()` and here, so that we only have a single reference to the environment variable. Is there a reason not to do so?